components/infobox/wikis/chess/infobox_league_custom.lua

```diff
@@ -74,7 +78,16 @@ function CustomInjector:parse(id, widgets)
 			Cell{name = 'Restrictions', content = caller:createRestrictionsCell(args.restrictions)}
 		)
 	elseif id == 'gamesettings' then
-		return {Cell{name = 'Variant', content = {caller.data.mode}}}
+		table.insert(
+			widgets,
+			Cell{name = 'Time Control', content = {caller.data.mode}}
+		)
+		if caller.data.game ~= GAMES['chess'] then
+			table.insert(
+				widgets,
+				Cell{name = 'Variant', content = {caller.data.game}}
```

Comment:
use module:Game for building the display

Reply:
> use module:Game for building the display

Can you provide an example of this? I couldn't find one.